### ISSUE TITLE
Jabref hangs on importing reference with en dash in page range

### ISSUE THREAD

**Issue by isaactpetersen** [JabRef/jabref#10169]
### JabRef version

5.9 (latest release)

### Operating system

Windows

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

The import window hangs when trying to import the a reference with an en dash in the page range. For example, the following .txt file throws an error:

```
@article{
   author = {Kriegman, Lois S. and Kriegman, George},
   title = {The PaTE report: A new psychodynamic and therapeutic evaluative procedure},
   journal = {The Psychiatric Quarterly},
   volume = {39},
   number = {1},
   pages = {646–674},
   doi = {10.1007/BF01569493},
   url = {https://doi.org/10.1007/BF01569493},
   year = {1965}
}
```

However, it works fine, when I change the en dash to two hyphens:

```
@article{
   author = {Kriegman, Lois S. and Kriegman, George},
   title = {The PaTE report: A new psychodynamic and therapeutic evaluative procedure},
   journal = {The Psychiatric Quarterly},
   volume = {39},
   number = {1},
   pages = {646--674},
   doi = {10.1007/BF01569493},
   url = {https://doi.org/10.1007/BF01569493},
   year = {1965}
}
```

**Comment by Siedlerchr** [comment]
I tested several ways of importing, but I could not reproduce your issue, works fine for me, using latest dev version on mac. 
Directly pasting of the maintable -> fine
File Import (new/current library) -> Works fine as well
Do you have any save actions defined? (Library properties -> Saving -> SAve actions?)

**Comment by isaactpetersen** [comment]
Yes, good catch. Here are my save actions. It includes "Normalize page numbers" (in addition to a few others, see attached image).
![image](https://github.com/JabRef/jabref/assets/3749025/dd55f5fc-9248-4bbb-8f60-ea3010d94734)
